scalding-core/src/main/scala/com/twitter/scalding/Execution.scala

```diff
@@ -603,11 +604,11 @@ object Execution {
    * but with proof that pipe matches sink
    */
 
-  sealed trait ToWrite
-  object ToWrite {
-    final case class Force[T](pipe: TypedPipe[T]) extends ToWrite
-    final case class ToIterable[T](pipe: TypedPipe[T]) extends ToWrite
-    final case class SimpleWrite[T](pipe: TypedPipe[T], sink: TypedSink[T]) extends ToWrite
+  sealed trait ToWrite extends Serializable
+  object ToWrite extends Serializable {
+    final case class Force[T](@transient pipe: TypedPipe[T]) extends ToWrite
+    final case class ToIterable[T](@transient pipe: TypedPipe[T]) extends ToWrite
+    final case class SimpleWrite[T](@transient pipe: TypedPipe[T], @transient sink: TypedSink[T]) extends ToWrite
```

Comment:
It might be worth documenting (here or in `KryoHadoop.scala`) that our strategy for serializing captures pipes/executions is to skip these fields (so they're initialized to `null`) and to explicitly serialize them as `null`. At least, I think that's the strategy we're using here, right?

Reply:
yeah, that's a good idea, since they are only used in planning, it should be safe, as the note in KryoHadoop says, but copying here is probably good too.